src/graphql/admin/mongoose_graphql_gdpr_admin_query.erl

```diff
@@ -0,0 +1,23 @@
+-module(mongoose_graphql_gdpr_admin_query).
+-behaviour(mongoose_graphql).
+
+-export([execute/4]).
+
+-import(mongoose_graphql_helper, [make_error/2]).
+
+-ignore_xref([execute/4]).
+
+-include("../mongoose_graphql_types.hrl").
+-include("mongoose.hrl").
+-include("jlib.hrl").
+
+execute(_Ctx, gdpr, <<"retrievePersonalData">>, #{<<"username">> := User, <<"domain">> := Domain,
+                                                  <<"resultFilepath">> := FilePath}) ->
+    try gdpr_api:retrieve_all(User, Domain, FilePath) of
+        ok -> {ok, "Data retrieved"};
+        {error, String} -> make_error({user_does_not_exist_error, String},
```

Comment:
It is a bit strange that the message can be any string(in theory), but we always tag it with `user_does_not_exist_error`. This is a bit counter-intuitive. Maybe the function could return that atom, and we could add the string here?